src/features/decks/ui/addDeck/AddDeck.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { action } from '@storybook/addon-actions'
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { AddDeck, AddDeckFormValues, AddDeckProps } from './'
+
+const meta: Meta<typeof AddDeck> = {
+  title: 'Decks/AddDeck',
+  component: AddDeck,
+  tags: ['autodocs'],
+}
+
+export default meta
+type Story = StoryObj<typeof meta>
+
+const CreateDeckWithHooks = (args: AddDeckProps) => {
+  return (
+    <>
+      <AddDeck variant={args.variant} values={args.values} onSubmit={args.onSubmit} />
```

Comment:
we may not use React.Fragment in this case

Reply:
Remove React.Fragment